sql-core/src/main/java/io/cdap/sql/core/SQLTable.java

```diff
@@ -0,0 +1,54 @@
+package io.cdap.sql.core;
+
+import io.cdap.sql.core.enums.SQLReadableType;
+import io.cdap.sql.core.interfaces.Aliasable;
+import io.cdap.sql.core.interfaces.SQLReadable;
+
+public class SQLTable implements SQLReadable, Aliasable {
+  public SQLTable(String project, String db, String table, String alias) {
```

Comment:
annotate db as `@Nullable`.

In general, we annotate everything that can be null and everything else is assumed to be non-null

Reply:
Sounds good, fixed